### PR TITLE
story and rules have been added with their titles and style

### PR DESCRIPTION
**Pull request by pagouillart**

### CODE REVIEW

**Review by Cyrildab — APPROVED** [review]
Nice !

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx:1]: An h2 tag cannot be nested within a p tag

Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx:1]: so the br tag is unnecessary

File: frontend/src/components/HomePage/ContainerStoryRules/Story/Story.jsx
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Story/Story.jsx:1]: Same with other comment

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx
Comment (pagouillart) [reply frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx:1]: done

Comment (pagouillart) [reply frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.jsx:1]: done :)

File: frontend/src/components/HomePage/ContainerStoryRules/Story/Story.jsx
Comment (pagouillart) [reply frontend/src/components/HomePage/ContainerStoryRules/Story/Story.jsx:1]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**